lib/adbc_column.ex

```diff
@@ -1386,5 +1386,19 @@ defmodule Adbc.Column do
 
   def to_list(%Adbc.Column{data: data, type: :list}), do: Enum.map(data, &to_list/1)
 
+  def to_list(%Adbc.Column{data: data, type: :struct}) do
+    struct_to_list(data)
+  end
+
+  def to_list(%Adbc.Column{data: data, type: {:struct, _columns}}) do
```

Comment:
According to the typespec above, it can only be `type: :struct`, never {:struct, _columns}. Is this clause necessary?